documentation/src/main/asciidoc/stories/assembly_virtual_threads.adoc

```diff
@@ -0,0 +1,20 @@
+[id='virtual-threads-{context}']
```

Comment:
I think we need to mention this change in `changes.adoc` so that users are aware that we have changed the default behaviour.

Reply:
Hey @pruivo, do you think it is worth disabling virtual threads in the code if the CPU count is low?

Reply:
Ok, just wanted to check. Thanks